templates/experiments/single_experiment_home.html

```diff
@@ -212,54 +212,244 @@ <h3 class="font-bold text-lg">Parent Experiments:
+                    <i class="fa-solid fa-spinner fa-spin"></i>
                   </div>
-                  {% if perms.annotations.view_customtaggeditem %}
-                    <div class="w-72 my-2">
-                      <select x-model="tags" id="tag-multiselect-filter" name="tags" multiple
-                              placeholder="Filter tags..."
-                              autocomplete="off">
-                        <option value="">Filter tags...</option>
-                        {% for tag in available_tags %}
-                          <option value="{{ tag.name }}">{{ tag.name }}</option>
-                        {% endfor %}
-                      </select>
-                    </div>
-                  {% endif %}
-                  <div class="form-control">
-                    <label class="label cursor-pointer">
-                      <span class="label-text mr-2">Include API Sessions</span>
-                      <input type="checkbox" name="show-all" class="toggle" />
-                    </label>
+
+                  <!-- Clear filters button -->
+                  <button
+                    x-show="getActiveFilterCount() > 0 && !filterData.loading"
+                    @click="clearAllFilters()"
+                    class="btn btn-square btn-sm btn-ghost"
+                    title="Clear all filters" x-cloak>
+                    x
+                  </button>
+                </div>
+
+              <!-- Filters Panel -->
+                <div
+                  x-show="filterData.showFilters"
+                  class="absolute left-0 mt-2 p-4 bg-white border border-gray-200 rounded-lg shadow-lg z-10 w-fit"
+                >
+                  <div class="space-y-3">
+                    <template x-for="(filter, index) in filterData.filters" :key="index">
+                      <div class="flex items-center gap-3">
+                      <!-- Where/AND Label -->
+                        <div class="text-gray-500 font-medium w-16">
+                          <span x-text="index === 0 ? 'Where' : 'AND'"></span>
+                        </div>
+                      <!-- Column Selector -->
+                        <div class="relative py-2">
+                          <div class="dropdown dropdown-bottom">
+                            <label
+                              tabindex="0"
+                              class="block w-40 px-3 border border-gray-300 rounded-lg bg-white text-sm cursor-pointer flex justify-between items-center"
+                              style="height: 2rem;"
+                            >
+                              <span x-text="filter.column ? filterData.columns[filter.column].label : 'Column'"></span>
+                              <i class="fa-solid fa-caret-down fa-sm"></i>
+                            </label>
+                            <ul tabindex="0" class="dropdown-content z-[1] menu p-2 bg-base-100 rounded-box border border-gray-300 w-40">
```

Comment:
styling:
```suggestion
                            <ul tabindex="0" class="dropdown-content z-[1] mt-1 menu p-2 bg-base-100 rounded-lg border border-gray-300 w-40">
```

(apply the same to the operator selector)